src/ListView/CreatePage/CreatePageForm.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from "react";
+import { colorTextErrorOnLight } from "@sailthru/design-tokens";
+import {
+  FormGroup,
+  Radio,
+  RadioGroup,
+  Subtext,
+} from "@sailthru/stui-components";
+import { Fieldset, Label, Input } from "@sailthru/stui-elements";
+
+function CreatePageForm({
+  name = "",
+  onChange,
+  isDuplicateName,
+  isNameChanged,
+}) {
+  const [namingError, setNamingError] = useState("");
+  const [optoutNamingError, setOptoutNamingError] = useState("");
+
+  // function to check for empty, alphanumeric, and 'oc' page names
+  const handleNamingError = function () {
+    setNamingError("");
+    setOptoutNamingError("");
+    if (name.length < 1) {
+      setNamingError("Please enter a page name.");
+      return;
+    }
+    if (name === "oc") {
+      setNamingError(
+        "The name 'oc' is a reserved page name. Please use a different page name."
+      );
+      return;
+    }
+    if (name === "optout") {
+      setOptoutNamingError(
+        "The page name 'optout' requires 'User Management' as the category. Please select 'User Management' below."
+      );
+      return;
+    }
+    const alphanumericRegex = /^[a-zA-Z0-9]+$/;
+    const nameWithoutDashes = name.replace(new RegExp("-", "g"), "");
+    if (!nameWithoutDashes.match(alphanumericRegex)) {
+      setNamingError(
+        "Page names can only contain alphanumeric characters and dashes."
```

Comment:
nit-pick: Trying to better match the other error message:
```suggestion
        "The page name must only contain alphanumeric characters and dashes. Please use a different page name."
```

Reply:
updated